Userland/Services/WebDriver/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+serenity_component(
+    WebDriver
+    TARGETS WebDriver
+)
+
+set(SOURCES
+    BrowserConnection.cpp
+    Client.cpp
+    Session.cpp
+    main.cpp
+)
+
+set(GENERATED_SOURCES
+    ../../Applications/Browser/WebDriverSessionClientEndpoint.h
```

Comment:
This path ain't gonna work for the Lagom build. We build services by add_subdirectory rather than by adding a separate build rule